stores/Batteries/BatteriesSagas.ts

```diff
@@ -0,0 +1,29 @@
+import BaseSaga from '@/utils/BaseSaga';
+import BatteriesActions from './BatteriesActions';
+import { call, delay, put } from 'redux-saga/effects';
+import { Battery } from './BatteriesTypes';
+import { BatteriesService } from './BatteriesService';
+
+export default class BatteriesSagas extends BaseSaga {
+  *[BatteriesActions.POLL_BATTERIES]() {
+    while (true) {
+      try {
+        const batteries: Battery[] = yield call(BatteriesService.getBatteries);
+        if (batteries.length > 0) {
+          const latestBattery = batteries[batteries.length - 1];
+          yield put({
+            type: BatteriesActions.REQUEST_BATTERIES_SUCCESS,
+            payload: latestBattery,
```

Comment:
One small comment - it should be okay for us to store all the battery data here and not just the lastest since we want to visualize the `voltage` or `current` overtime